great_expectations/_docs_decorators.py

```diff
@@ -60,6 +69,23 @@ def add(self, func: F) -> None:
             logger.exception(f"Could not add this function to the public API list: {func}")
             raise
 
+    def _add_to_class_registry(self, func: F) -> None:
+        if isinstance(func, type):
+            key = f"{func.__module__}.{func.__qualname__}"
+            self._class_registry[key].add(self.CLASS_DEFINITION)
```

Comment:
How do we know this is a class and not a top level function? I haven't yet looked at how this is used, so does that matter?

Reply:
`type` denotes a class here